internal/engine/eval/vulncheck/review_test.go

```diff
@@ -542,7 +538,11 @@ func TestCommitStatusPrHandlerWithVulnerabilities(t *testing.T) {
 					Path:      github.String(dep.File.Name),
 					StartLine: github.Int(1),
 					Line:      github.Int(4),
-					Body:      github.String(expCommentBody),
+					Body: github.String("```suggestion\n\n" +
+						"  \"version\": \"0.6.0\",\n" +
+						"  \"resolved\": \"https://registry.npmjs.org/mongodb/-/mongodb-0.6.0.tgz\",\n" +
+						"  \"integrity\": \"sha512-+1+2+3+4+5+6+7+8+9+0\",\n" +
+						"```\n"),
```

Comment:
It may be easier to compose this as:

```suggestion
					Body: github.String("```suggestion"+`

  "version": "0.6.0",
  "resolved": "https://registry.npmjs.org/mongodb/-/mongodb-0.6.0.tgz",
  "integrity": "sha512-+1+2+3+4+5+6+7+8+9+0
`+"```\n"),
```